.github/workflows/benchmark-history.yml

```diff
@@ -40,4 +40,26 @@ jobs:
       run: make freeze
     - name: Run benchmarks
       run: |
-        .github/scripts/benchmark-history.sh
\ No newline at end of file
+        .github/scripts/benchmark-history.sh benchmarks/tabular
+
+  benchmark-history-vision:
```

Comment:
Why separate to different jobs..? Can we somehow do it in the same script?

Reply:
but why? it looks better like that
you can do everything in one big job

Reply:
Because: 
1. It will require more resources from GitHub
2. I don't know how it will look in history. did you check?